src/services/auth.js

```diff
@@ -62,38 +62,30 @@ export const changePassword = async (id, data) => {
 export const forgotPasswordRequest = async data => {
   try {
     const response = await httpService.post({ url: links.resetRoute, data });
-    return response;
+    return response.data;
   } catch (e) {
     return {
       status: 400,
-      data: 'Bad email',
+      message: 'Bad email',
     };
```

Comment:
You throwing error from the backend and don't catch it here and just hardcoded the error message. Look how we catch errors on `loginRequest`.

Reply:
Done